core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -442,7 +442,7 @@
       "The maximum number of files a compaction will open"),
   TSERV_COMPACTION_SERVICE_ROOT_EXECUTORS(
       "tserver.compaction.major.service.root.planner.opts.executors",
-      "[{'name':'small','maxSize':'32M','numThreads':1},"
+      "[{'name':'small','maxSize':'32M','numThreads':1},".replaceAll("'", "\"")
           + "{'name':'huge','numThreads':1}]".replaceAll("'", "\""),
```

Comment:
Could do this instead:

```suggestion
      ("[{'name':'small','maxSize':'32M','numThreads':1},"
          + "{'name':'huge','numThreads':1}]").replaceAll("'", "\""),
```

Reply:
Oh yeah good idea.